shell/common/api/electron_api_native_image.cc

```diff
@@ -62,6 +62,9 @@ base::FilePath NormalizePath(const base::FilePath& path) {
     return path;
   }
 
+  // FIXME(zcbenz): MakeAbsoluteFilePath is a blocking call, figure out how we
+  // we can remove the |allow_blocking| here.
+  base::ScopedAllowBlockingForTesting allow_blocking;
```

Comment:
```
{
  base::ThreadRestrictions::ScopedAllowIO allow_io;
  base::FilePath absolute_path = MakeAbsoluteFilePath(path);
}
```

Instead of the testing api.